stores/sql/mysql/main.go

```diff
@@ -862,6 +811,62 @@ func (tx *MainDatabaseTx) UpdateBucketPolicy(ctx context.Context, bucket string,
 	return ssql.UpdateBucketPolicy(ctx, tx, bucket, bp)
 }
 
+func (tx *MainDatabaseTx) UpdateContractSet(ctx context.Context, name string, toAdd, toRemove []types.FileContractID) error {
+	res, err := tx.Exec(ctx, "INSERT INTO contract_sets (name) VALUES (?) ON DUPLICATE KEY UPDATE id = last_insert_id(id)", name)
+	if err != nil {
+		return fmt.Errorf("failed to insert contract set: %w", err)
+	}
+
+	// if no changes are needed, return after creating the set
```

Comment:
Hm, still think a 400 is appropriate if both are empty.

Reply:
I think it's nice to have a way of creating a contract set without contracts. Even if it's just for some UI that let's you create an empty set before switching tabs and selecting contracts to the set.